examples/Atlas/+atlasControllers/QPInputConstantHeight.m

```diff
@@ -57,6 +58,8 @@
       obj.body_motion_data = struct('body_id', {},... % 3 d
                             'ts', {},... % 6 d
                             'coefs', {}); % 4 * 6 * 3 d
+      obj.body_wrench_data = struct('body_id', {},...
+                                    'wrench', {});
```

Comment:
It's a wrench in body frame, yeah. So the angular part is the torque about the origin of the body frame, where the 3 coordinates of the torque vector correspond to magnitudes along the axes of that body frame. Likewise, the force part is a free vector expressed in body frame (origin doesn't matter for force, just the orientation of the frame).
As in definition 2.13 [here](http://eprints.eemcs.utwente.nl/1622/01/vduindamPhDthesis.pdf).